F/FLINT/build_tarballs.jl

```diff
@@ -25,7 +25,7 @@ using BinaryBuilder, Pkg
 # coordinated with corresponding changes to Singular_jll.jl, Nemo.jl and polymake_jll.jl
 # and possibly other packages.
 name = "FLINT"
-upstream_version = v"3.2.1"
+upstream_version = v"3.2.2"
 version_offset = v"0.0.0"
```

Comment:
```suggestion
version_offset = v"0.0.1"
```
maybe bump the offset to mark that this is not *pure* 3.2.2, but includes some further change (i.e. the patch you included)